### DISCUSSION TITLE
Error with the requirements in section 9

### DISCUSSION THREAD

**Discussion by rootdobos** [mrdbourke/pytorch-deep-learning#730] [Q&A]
Hi Daniel,

I copied the requirements for the food vision mini requirements.txt from the learnpytorch.io and when I deployed it I got a runtime error. I figured out that the problem was the gradio version on the learnpytorch.io changing it from 3.1.4 to 3.14.0 fixed the issue.

**Comment by shisirkha** [comment]
This might be a documentation issue note this as an issue.
